scripts/services/rewrite-services-json.js

```diff
@@ -0,0 +1,78 @@
+const { promises: fs } = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const { checkSVG } = require('./check-svg');
+
+/**
+ * Checks that all services have valid SVG icons.
+ *
+ * @param {Array<object>} servicesArray - An array of service data objects.
+ * @returns {boolean} True if all services have valid SVG icons.
+ * @throws {Error} If an error(s) occurred during SVG validation, an error is thrown.
+ */
+const validateSvgIcons = (servicesArray) => {
```

Comment:
this function can be moved to the `check-svg.js`, and after that it would be better to rename
`check-svg.js` -> `validate-svg-icons.js`